linera-sdk/src/abis/fungible.rs

```diff
@@ -32,14 +32,14 @@ impl ServiceAbi for FungibleTokenAbi {
 pub enum Operation {
     /// Requests an account balance.
     Balance {
-        /// Owner to query the balance for
+        /// MultiAddress to query the balance for
```

Comment:
```suggestion
        /// `MultiAddress` to query the balance for
```

Or "Owner", or "Address" (both without backticks)?